packages/react-table/src/components/Table/utils/decorators/treeRow.tsx

```diff
@@ -8,92 +8,90 @@ import { Checkbox } from '@patternfly/react-core/dist/esm/components/Checkbox';
 import AngleDownIcon from '@patternfly/react-icons/dist/esm/icons/angle-down-icon';
 import EllipsisHIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-h-icon';
 
-export const treeRow = (
-  onCollapse: OnTreeRowCollapse,
-  onCheckChange?: OnCheckChange,
-  onToggleRowDetails?: OnToggleRowDetails
-) => (value: IFormatterValueType, { rowIndex, rowData }: IExtra) => {
-  const {
-    isExpanded,
-    isDetailsExpanded,
-    'aria-level': level,
-    'aria-setsize': setsize,
-    toggleAriaLabel,
-    checkAriaLabel,
-    showDetailsAriaLabel,
-    isChecked,
-    checkboxId,
-    icon
-  } = rowData.props;
-  const content = value.title || value;
-  const text = (
-    <div className={css(stylesTreeView.tableTreeViewText)} key="tree-view-text">
-      {icon && (
-        <span className={css(stylesTreeView.tableTreeViewIcon)} key="tree-view-text-icon">
-          {icon}
+export const treeRow =
+  (onCollapse: OnTreeRowCollapse, onCheckChange?: OnCheckChange, onToggleRowDetails?: OnToggleRowDetails) =>
+  (value: IFormatterValueType, { rowIndex, rowData }: IExtra) => {
+    const {
+      isExpanded,
+      isDetailsExpanded,
+      'aria-level': level,
+      'aria-setsize': setsize,
+      toggleAriaLabel,
+      checkAriaLabel,
+      showDetailsAriaLabel,
+      isChecked,
+      checkboxId,
+      icon
+    } = rowData.props;
+    const content = value.title || value;
+    const text = (
+      <div className={css(stylesTreeView.tableTreeViewText)} key="tree-view-text">
+        {icon && (
+          <span className={css(stylesTreeView.tableTreeViewIcon)} key="tree-view-text-icon">
+            {icon}
+          </span>
+        )}
+        <span className="pf-v5-c-table__text" key="table-text">
+          {content as React.ReactNode}
         </span>
-      )}
-      <span className="pf-c-table__text" key="table-text">
-        {content as React.ReactNode}
-      </span>
-    </div>
-  );
-  const onChange = (isChecked: boolean, event: React.FormEvent<HTMLInputElement>) => {
-    onCheckChange(event, isChecked, rowIndex, content as React.ReactNode, rowData);
+      </div>
+    );
+    const onChange = (isChecked: boolean, event: React.FormEvent<HTMLInputElement>) => {
+      onCheckChange(event, isChecked, rowIndex, content as React.ReactNode, rowData);
+    };
+    return {
+      component: 'th',
+      className: 'pf-c-table__tree-view-title-cell',
```

Comment:
Missing pf-v5-c.... ?

Reply:
Made this update alongside everything else